app/views/finders/_option_select_facet.html.erb

```diff
@@ -1,8 +1,8 @@
-<% add_app_component_stylesheet("option-select") %>
+<% add_gem_component_stylesheet("option-select") %>
 <% add_gem_component_stylesheet("checkboxes") %>
 
 <% cache_if option_select_facet.cacheable?, option_select_facet.cache_key do %>
-  <%= render partial: 'components/option_select', locals: {
+  <%= render partial: 'govuk_publishing_components/components/option_select', locals: {
```

Comment:
The `finder-frontend` `option-select` had these GA4 attributes on the parent`<div>`:

```HTML
data-ga4-change-category="update-filter checkbox" data-ga4-filter-parent
```

and on the child `<div>`: `data: { ga4_section: title }` 

Without these the GA4 tracking won't work for this component.

You should be able to wrap the render in just one `<div>` with the attributes, e.g. 

```HTML
<div data-ga4-change-category="update-filter checkbox" data-ga4-filter-parent data-ga4-section="[The input label]">
``` 

And then when you add/remove a checkbox, there should be an event in the dataLayer.

I guess we have lost the functionality to test it though, unless you have a way to test this?

Thanks :+1:

Reply:
Thanks for spotting this @AshGDS. I've applied your fix (in a separate PR) and raised another [PR in the gem](https://github.com/alphagov/govuk_publishing_components/pull/3625) as these attributes had been incorrectly ported across when I moved the component there.

In terms of testing we might be able to check that the overall tracking for the option selects on this page is working in a feature test, but I think the JS tests for the filter parent/set index stuff should also be able to cover off a general test to check that the functionality works in principle (if they don't already - I suspect they do, I remember your tests being pretty thorough). I'll put a tech debt card on our board to investigate this.

Reply:
@andysellick Thanks :+1: